devicetypes/smartthings/smartpower-outlet.src/README.md

```diff
@@ -0,0 +1,37 @@
+# SmartPower Outlet
+
+
+
+Works with: 
+
+* [Samsung SmartPower Outlet](https://shop.smartthings.com/#!/products/smartpower-outlet)
+
+## Table of contents
+
+* [Capabilities](#capabilities)
+* [Health]($health)
```

Comment:
These `Health` & `Battery` URLs gives 404. It needs to be `Device Health` or the Table of Content needs to change.